_dashboards/quickstart-dashboards.md

```diff
@@ -11,9 +11,6 @@ redirect_from:
 
 This quickstart guide covers the core concepts that you need to understand to get started with OpenSearch Dashboards. You'll learn how to:
 
-- Add sample data.
-- Explore and inspect data with **Discover**.
-- Visualize data with **Dashboard**.
 - Add sample data.
 - Explore and inspect data with **Discover**.
 - Visualize data with **Dashboard**.
```

Comment:
What is Dashboard? Can we just say Visualize data? I don't think the where is as important as the what and why.

Reply:
Dashboard is the application on the UI.

Reply:
Change made.